app/containers/Forms/ProductRowIdField.tsx

```diff
@@ -23,25 +24,69 @@ export const ProductRowIdFieldComponent: React.FunctionComponent<Props> = props
       ...props,
       schema: {
         ...props.schema,
-        enum: props.query.allProducts.edges.map(({node}) => node.rowId),
-        enumNames: props.query.allProducts.edges.map(({node}) => node.name)
+        enum: props.query.active.edges.map(({node}) => node.rowId),
+        enumNames: props.query.active.edges.map(({node}) => node.name)
       },
       query: undefined
     }),
     [props]
   );
 
-  return <props.registry.fields.StringField {...fieldProps} />;
+  const activeProductIds = props.query.active.edges.map(({node}) => node.rowId);
+  const inactiveProductIds = [
+    ...props.query.archived.edges.map(({node}) => node.rowId),
+    ...props.query.deprecated.edges.map(({node}) => node.rowId)
+  ];
+  const inactiveProductNames = [
+    ...props.query.archived.edges.map(({node}) => node.name),
+    ...props.query.deprecated.edges.map(({node}) => node.name)
+  ];
```

Comment:
You should make use of `useMemo` to avoid iterating on all the product lists on every render

Reply:
Also do you mind updating that component's documentation above while you're at it. It's an out-of-data copy-paste, so it's not accurate at all